lib/common.sh

```diff
@@ -159,6 +159,12 @@ export IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironi
 export IRONIC_DATA_DIR="$WORKING_DIR/ironic"
 export IRONIC_IMAGE_DIR="$IRONIC_DATA_DIR/html/images"
 export IRONIC_KEEPALIVED_IMAGE=${IRONIC_KEEPALIVED_IMAGE:-"quay.io/metal3-io/keepalived"}
+# Option to use reverse proxy on ironic-inspector
+export INSPECTOR_REVERSE_PROXY_SETUP=${INSPECTOR_REVERSE_PROXY_SETUP:-"false"}
```

Comment:
I might be missing the bits, but are not we supposed to have this option enabled by default? AFAIK, `IRONIC_TLS_SETUP` is used by default.
Also, vars.md needs to be updated with new var.

Reply:
I was not fully clear in prev. comment, maybe. Is it the case that we will have 2 options depending on `IRONIC_TLS_SETUP`?:

-  `IRONIC_TLS_SETUP=false` ==> `INSPECTOR_REVERSE_PROXY_SETUP=false`;

- `IRONIC_TLS_SETUP=true` ==> `INSPECTOR_REVERSE_PROXY_SETUP=false/true`;